app/models/pet.rb

```diff
@@ -9,10 +9,14 @@ def self.sort_by_status
   end
 
   def self.adoptable
-    Pet.where(status: "Adoptable")
+    Pet.where(status: "Adoptable").order(:id)
```

Comment:
no idea what's causing these to sometimes shuffle order, added these as a quick fix